x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -109,6 +110,8 @@ const NO_ROW_RENDER: RowRenderer[] = [];
 
 const trailingControlColumns: never[] = [];
 
+const O11Y_ALERT_CONSUMER = [ALERTS_CONSUMERS.APM, ALERTS_CONSUMERS.LOGS];
```

Comment:
We need to also include synthetics.

Reply:
Consider it more like an example for you, you can add what you need later on. ofc I will add Synthetics for now.